packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -0,0 +1,203 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { stringToBuffer } from "@fluid-internal/client-utils";
+import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct/internal";
+import {
+	AttachState,
+	type IRuntimeFactory,
+} from "@fluidframework/container-definitions/internal";
+import { loadContainerRuntime } from "@fluidframework/container-runtime/internal";
+import type { IFluidHandle } from "@fluidframework/core-interfaces";
+import type { FluidObject } from "@fluidframework/core-interfaces";
+import { assert, Lazy, LazyPromise } from "@fluidframework/core-utils/internal";
+import type { IChannel } from "@fluidframework/datastore-definitions/internal";
+import type { IDataStore } from "@fluidframework/runtime-definitions/internal";
+import { toFluidHandleInternal } from "@fluidframework/runtime-utils/internal";
+
+import { ddsModelMap } from "./ddsModels";
+
+export class StressDataObject extends DataObject {
+	public static readonly factory = new Lazy(() => {
+		const factory = new DataObjectFactory(
+			"StressDataObject",
+			StressDataObject,
+			[...ddsModelMap.values()].map((v) => v.factory),
+			{},
+			[["StressDataObject", new LazyPromise(() => factory)]],
+		);
+		return factory;
+	});
+
+	get StressDataObject() {
+		return this;
+	}
+
+	protected _globalObjects: Record<
```

Comment:
so this object is a bit weird, but essential to how all of this works. 

This object always comes from the default datastore, so there is only a single instance for the entire instance of the container. We then register all container scope object here, blobs, alias, data stores. This provides a few capabilities:
1) find things to reference via handles
2) walk the whole container so we can do validation

two very important parts of this is it is in-memory only, as it contains both attached and detached things, and secondly it scope is to a single container instance, so we should never access globalObjects between container instances.

Reply:
This kind of thing frequently goes into some scoped portion of the fuzz test state object. Not sure if that ends up being easier explained, but e.g. I could imagine including all of these as fields on `Client` which get modified over time by the fuzz test's reducers. You could invert the responsibility of bookkeeping this state into the model if you wanted (so StressDataObjects take in a `onChildCreateRecursive: () => void` which the fuzz test implements by augmenting the fuzz test state).

Kind of indifferent on if that's actually better than the pattern you used here. I'd probably rename it to `containerScopedObjects()` and document with this comment, at which point it seems pretty reasonable.